modules/setting/f3.go

```diff
@@ -0,0 +1,24 @@
+// Copyright 2022 The Gitea Authors. All rights reserved.
+// Use of this source code is governed by a MIT-style
+// license that can be found in the LICENSE file.
+
+package setting
+
+import (
+	"code.gitea.io/gitea/modules/log"
+)
+
+// Friendly Forge Format (F3) settings
+var (
+	F3 = struct {
+		Enabled bool
+	}{
+		Enabled: true,
+	}
+)
+
+func newF3Service() {
+	if err := Cfg.Section("F3").MapTo(&F3); err != nil {
```

Comment:
all other sections are lowercase
```suggestion
	if err := Cfg.Section("f3").MapTo(&F3); err != nil {
```